Backend/Routes/signup.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express')
+const router = express.Router()
+const { PrismaClient } = require('@prisma/client')
+const prisma = new PrismaClient()
+const cors = require('cors');
+router.use(cors());
+
+router.get("/test", async (req, res) => {
```

Comment:
Is this in use? if it is just test use, shall we remove it from production code?

Reply:
It was for test use, thanks for pointing out! I have removed it.